metadata.yaml

```diff
@@ -14,3 +14,6 @@ assumes:
 provides:
   ingress:
     interface: ingress
+
```

Comment:
Picky comment, I'd remove this line

Reply:
I removed all empty lines in the `metadata.yaml`, thanks!